### DISCUSSION TITLE
Face recognition workflow?

### DISCUSSION THREAD

**Discussion by IngwiePhoenix** [photoprism/photoprism#3027] [General]
Hello!

Due to my visual impairment, I have trouble seing images properly - let alone going through quite a backlog of them of taking photos with my phone for many years. Not particularily great ones - but they mean a lot to me.

I finally want to get them organized and sorted - but to do so, I need to make sure the features work properly. So before doing too much, I want to know how exactly the face recognition works and what the workflow is.

I imported a 200MB sample of my library - and even after two full re-scans, I see no entries in the "New Faces" category - only when I click the button does it show me a few images - but they are not grouped together as I would have expected them to be as they share a face.

So... Am I just doing something wrong? Anything I can do here?

Thank you and kind regards,
Ingwie

**Comment by graciousgrey** [comment]
The new people tab shows face clusters. If no clusters show up, the detected faces could not be grouped together. 
How many photos do you have imported? Recognition works best the more images there are of a person. 

For more information on facial recognition see: https://docs.photoprism.app/user-guide/organize/
